maquetado/index.html

```diff
@@ -11,23 +11,24 @@
 <body>
   <div class="book-container row">
     <img src="./assets/book-cover.png" class="book-cover">
+    <img src="./assets/book-badge.png" class="book-badge">
```

Comment:
I would set this as a `pseudo-element` (after or before) since it's a decorative asset.